project.clj

```diff
@@ -1,8 +1,15 @@
+(defn deploy-info
```

Comment:
@KevinCorcoran We should probably avoid pulling in any of the changes to this file except for the `:source-paths` and `:java-source-paths`.  We wouldn't want to have the project built and deployed to a non-Clojars repo.